### PR TITLE
chore: update script heuristics to find more meaningful repositories to ingest

### PR DESCRIPTION
**Pull request by sjungling**
also limiting to 500 by lang/topic.

### CODE REVIEW

File: repos.csv [hunk 1]
```diff
@@ -47644,3 +47644,986 @@ scmHost,repoName,repoBranch,mavenTool,gradleTool,jdkTool,repoStyle,repoBuildActi
 ,zzz40500/Tinker_imitator,master,,,,,,TRUE,Build uses Gradle 2.x
 ,zzz40500/android-shapeLoadingView,master,,,,,,TRUE,Gradle wrapper 3.3 is not supported
 ,zzzzbw/Fame,master,,,,,,,
+",11ty/eleventy,main,,,,,,,"
+",1backend/1backend,master,,,,,,,"
+",1j01/jspaint,master,,,,,,,"
```
Comment (timtebeek) [inline repos.csv:47649]: We'll want to remove the wrapping quotes here.

Comment (sjungling) [reply repos.csv]: yep, removing…

Comment (sjungling) [reply repos.csv]: spotted that _almost_ right away

**Review by timtebeek — APPROVED** [review]
Good to see this put to more use!

**Comment by timtebeek** [comment]
Had to do a quick fix on the job that would have otherwise triggered a downstream refresh of the Jenkins jobs:
https://github.com/moderneinc/jenkins-ingest/commit/b4c6fe781a63130133dc71e2b9fdcb060d54d3cb
I see you've also triggered that job manually already; nice!

**Comment by sjungling** [comment]
@timtebeek oooh, so normally it'll auto-run on commit to `main`? but can also be manually trigged via the action in  `moderne-cli` ?

**Comment by timtebeek** [comment]
Yup! Any changes made to repos.csv trigger a workflow that updates the Jenkins jobs; it even does a bit of foo to extract a prefix if it recognizes the commit message format:
https://github.com/moderneinc/jenkins-ingest/blob/b4c6fe781a63130133dc71e2b9fdcb060d54d3cb/.github/workflows/updateJenkins.yaml#L18-L24

**Pull request merged**